### PR TITLE
[Remove Stream/Optional usage] /LPS-174240 Remove_usages_of_Stream_usages_for_portal-store-azure

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

This is for ticket https://issues.liferay.com/browse/LPS-174240.
Thanks for checking.

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:help

**Comment by Kvale1733** [comment]
ci:test:dl-store

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java [hunk 1]
```diff
@@ -37,7 +37,6 @@
 
 import com.liferay.document.library.kernel.exception.NoSuchFileException;
 import com.liferay.document.library.kernel.store.Store;
-import com.liferay.document.library.kernel.util.DLUtil;
```
Comment (tinatian) [inline modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java:40]: @jeffwu0724 Why the usage os DLUtil is removed?

**Comment by jeffwu0724** [comment]
added the `comparator DLUtil::compareVersions` in` Collections.sort(fileVersions, DLUtil::compareVersions)`, will rerun the test

**Comment by jeffwu0724** [comment]
ci:test:dl-store

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java [hunk 2]
```diff
@@ -208,14 +208,14 @@ public String[] getFileNames(
...
-				companyId, repositoryId, blobItem.getName())
-		).toArray(
-			String[]::new
-		);
+		for (BlobItem blobItem : pagedIterable) {
+			fileNames.add(
+				_getFileName(companyId, repositoryId, blobItem.getName()));
+		}
+
+		return (String[])fileNames.toArray();
```
Comment (tinatian) [inline modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java:218]: @jeffwu0724 This should be able to use TransformUtil

**Comment by tinatian** [comment]
@jeffwu0724 Please see my comment and resend.

Tina

**Pull request closed without merging**